src/components/ui/TopComponent.tsx

```diff
@@ -8,7 +8,7 @@ const RootContainer = styled.div`
     padding: 50px 122px 50px 122px;
 
     max-width: 1440px;
-    margin: 0px auto;
+    margin 0px;
```

Comment:
Check: this is invalid css syntax, I believe. Is it being honoured but missing the `:`, or is it being silently ignored by the browser?

Reply:
must not have my styled-components extension wired up as it should have caught this. in any case, checking in devtools it looks like the browser recovers from this syntax error. that said, I've fixed it.